src/rust/ide/lib/view/src/graph_editor/component/documentation_view.rs

```diff
@@ -0,0 +1,159 @@
+//! Documentation view visualization generating and presenting Enso Documentation under
```

Comment:
Please move this file to `src/graph_editor/builtin/visualization/native/documentation_view` instead. Of course, all related files should be relocated as well. Visualization view is a built-in visualization, so it should be located there IMO.

Reply:
Okk, what's funny is that i've actually started with having it there, but then i looked at task spec and moved it here